src/schemas/execute.yaml

```diff
@@ -168,7 +166,7 @@ Withdrawal:
       $ref: '#/components/schemas/uint64'
 EthSimulateResult:
   title: Full results of multi call
-  type: object
+  type: array
```

Comment:
This should be an `object`.  See the full proper fix for this issue in https://github.com/ethereum/execution-apis/pull/670/files, which we were waiting to merge until the CI bug was fixed but we could merge that now if we don't want to wait for the bug in the CI checker to get fixed.

Reply:
Ah, I think I see the problem.  This is a `oneOf` and one of the options is an array and the other is an object.  I'm not sure how to capture this in openrpc specs.  Do we just remove the `type` entirely?

Reply:
I think the object `EthSimulateBlockResultInvalid` is also wrong, but there are no tests for it so it doesn't fail the schema. AFAICT, invalid results fail with a JSON-RPC error, not a success response with `result` field.

Not sure what the expected value for it is.

Reply:
To express this you will need to use the `errors` field on the method object: https://spec.open-rpc.org/#method-object

Reply:
I've done this in #[d330c9b](https://github.com/ethereum/execution-apis/pull/675/commits/d330c9b412309cdd7ac22bbc2708b0ed4383c6b7)

PTAL